deployability/modules/testing/tests/test_agent/test_stop.py

```diff
@@ -71,7 +71,8 @@ def test_service(wazuh_params):
         GeneralComponentActions.component_stop(agent_params, 'wazuh-agent')
 
     for agent_names, agent_params in wazuh_params['agents'].items():
-        assert 'inactive' in GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent') or 'not running' in GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent'), logger.error(f'{agent_names} is still active by command')
+        status = GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent')
+        assert 'inactive' in status or 'Stopped' in status or 'StopPending' in status or 'not running' in status, logger.error(f'{agent_names} is still active by command')
```

Comment:
The variable `status` is a string. I think this expression is easier to read:

`status in ['inactive', 'Stopped', 'StopPending', 'not running']`

If you agree replace these expressions in the code.

Reply:
Done in  https://github.com/wazuh/wazuh-qa/pull/5310/commits/65ca9b4687cd5d98cb49b609aa5d1f9a662d666c